src/cript/api/api.py

```diff
@@ -111,6 +138,8 @@ def __init__(self, host: Union[str, None], token: [str, None]):
             You can find your personal token on the cript website at User > Security Settings.
             The user icon is in the top right.
             If `None` is specified, the token is inferred from the environment variable `CRIPT_TOKEN`.
+        config_file_path: Union[Path, str]
```

Comment:
We should clearly document which of the option takes precedent if multiple are specified.

Reply:
sure, as soon as we get that figured out I can add it in

Reply:
I think the way I wrote it with utils folder makes it very flexible so we can change up the order whenever we want